### PR TITLE
Added info on default_params

### PR DESCRIPTION
**Pull request by aoaleksandrowicz**
Added from the readme doc since more relevant here.

### CODE REVIEW

**Comment by aoaleksandrowicz** [comment]
Added info from the readme docs here since more relevant

File: README.md [hunk 1]
```diff
@@ -380,6 +380,8 @@ The `functions` field contains individual `function` entries for each UDF in you
 
 *Tip: You may use the same `class_file` with a different `function_name` and `default_params` to produce multiple UDFs from the same class.*
 
+*Note: The default_params value are set within the function’s library manifest entry. Unless specified, all the parameters specified in the Params section are mandatory. The validation checks that are applied to the Params in the function validate method mean that a missing parameter value will result in a failed validation (by throwing a “Missing <PARAM_NAME> parameter”). The validation result should be a user-friendly description of the validated function.
```
Comment (daverted) [inline README.md:383]: What was the context of this? Wondering why `default_params` is singled out. What do you think about simplifying this a little?

*Note: Unless specified, all the parameters listed above are mandatory. A missing parameter value will result in a failed validation by throwing a "Missing <PARAM_NAME> parameter".*

The last sentence doesn't seem related here but could go up at the top of this readme where we talk about the `validateInput` method:
> The validation result should be a user-friendly description of the validated function.

Comment (aoaleksandrowicz) [reply README.md:383]: Sounds good. I moved this here because we had it in the regular docs and it seemed out of context there. Thx

Comment (daverted) [reply README.md:383]: I've opened PR #5 with the changes discussed here.

**Pull request closed without merging**